playwright/test/concept.test.ts

```diff
@@ -165,28 +165,27 @@ test.describe('a Concept representing a Genre with works and images both about a
 
 test.describe('a Concept representing a Genre that is only used as a genre for both works and images', () => {
   test('has both works and image sections showing records in that genre', async ({
-    lithographsPage,
+    mohPage,
   }) => {
     // Both images and works sections exist
-    await expect(lithographsPage.imagesHeader).toBeVisible();
-    await expect(lithographsPage.worksHeader).toBeVisible();
+    await expect(mohPage.imagesHeader).toBeVisible();
+    await expect(mohPage.worksHeader).toBeVisible();
+
     // There are no tabs, because there is only one group within each of the two sections
-    await expect(lithographsPage.worksAboutTab).not.toBeVisible();
-    await expect(lithographsPage.worksInTab).not.toBeVisible();
-    await expect(lithographsPage.imagesAboutTab).not.toBeVisible();
-    await expect(lithographsPage.imagesInTab).not.toBeVisible();
+    await expect(mohPage.worksAboutTab).not.toBeVisible();
+    await expect(mohPage.worksInTab).not.toBeVisible();
+    await expect(mohPage.imagesAboutTab).not.toBeVisible();
+    await expect(mohPage.imagesInTab).not.toBeVisible();
 
-    // It has links to filtered searches
-    await expect(lithographsPage.allWorksLink).toHaveAttribute(
+    // It has links to filtered searches, (not using encodeURIComponent because the genre includes '+'")
```

Comment:
`encodeURIComponent` url encodes "+" as "%2B", which does not match what `toHaveAttribute` is returning for the href.